domain/HousingApi.ts

```diff
@@ -483,3 +489,6 @@ export interface EvidenceRequestResponse {
   id: string;
   createdAt: string;
 }
+export interface SimpleTypeResponse<T> {
```

Comment:
Doesn't look like this is used?

Reply:
it was when we're returning an calculated bedroom number from the api. I've left it in there because the Bedroom calculator endpoint returns this..